exercises/concept/calculator-conundrum/.docs/intoduction.md

```diff
@@ -0,0 +1,122 @@
+# Exception Handling in Java
+
+The Exception Handling in Java is one of the powerful mechanism to handle the runtime errors so that the normal flow of the application can be maintained.
+
+Good exception handling can handle errors and gracefully re-route the program to give the user still a positive experience.
+
+#### Why use Exception Handling with an example
+
+```java
+public static List<Player> getPlayers() throws IOException {
+    Path path = Paths.get("players.dat");
+    List<String> players = Files.readAllLines(path);
+
+    return players.stream()
+      .map(Player::new)
+      .collect(Collectors.toList());
+}
+```
```

Comment:
This is probably a poor example precisely because it does not actually handle any exceptions. It would be better if we wrapped this and handled the IOException, perhaps logging to stderr and returning an empty list?

Reply:
This example knowingly does not handle the exception to show an example of the conditions caused in cases like these.